vue/src/modules/setup/employees/EmployeeFormModal.vue

```diff
@@ -1,10 +1,16 @@
 <template>
   <ValidationObserver ref="form" v-slot="{ handleSubmit, invalid ,reset}">
     <b-modal
-        :visible="showModal" id="user-form" ref="modal" :title='$t(`Edit employee`)' @hidden="onHideModal" size="lg"
+        :visible="showModal" id="user-form" ref="modal" :title='$t(`Edit employee "{user}"`, {user: object.email})' @hidden="onHideModal" size="lg"
         @ok.prevent="handleSubmit(onSubmit)" :ok-disabled="loading" :ok-title="$t('Submit')" scrollable>
       <content-spinner :show="loading" :text="$t('Please wait...')" :fullscreen="true" class="h-100"/>
       <b-form @submit.prevent="handleSubmit(onSubmit)" novalidate>
+        <b-card header-tag="header" class="form-cards mb-2" body-class="pb-0">
+          <input-widget v-if="model.status" style="color: green" size="lg" type="checkbox"
```

Comment:
@skavt For these checkboxes try using class `text-danger` or `text-success` instead of styles. Also you can use one checkbox with dynamic class.